Source: written by Pierre Lejeune <superheron@gmail.com>
Upstream: no, Exherbo specific
Reason: Show "Exherbo Linux" in System Info

--- a/files/usr/share/cinnamon/cinnamon-settings/modules/cs_info.py	2015-01-19 16:45:53.000000000 +0100
+++ b/files/usr/share/cinnamon/cinnamon-settings/modules/cs_info.py	2015-02-14 23:22:45.294543401 +0100
@@ -92,6 +92,13 @@
     elif os.path.isfile("/etc/arch-release"):
         title = "Arch Linux"
         infos.append((_("Operating System"), title))
+    elif os.path.isfile("/etc/os-release"):
+        for line in open("/etc/os-release"):
+            (key, sep, value) = line.partition("=")
+            if key == "PRETTY_NAME":
+                title = value.strip('"\n')
+
+        infos.append((_("Operating System"), title + " (" + arch + ")"))
     else:
         infos.append((_("Operating System"), dname + " " + dversion +  " '" + dsuffix.title() + "' (" + arch + ")"))    
     if 'CINNAMON_VERSION' in os.environ:            
